### PR TITLE
March Madness post, Zach and Allen

### PR DESCRIPTION
**Pull request by zbay**
Hi, @eah13. Allen and I resolved our merge conflict, completed the bracket exercise, and wrote a post on it. Please start merging my pull requests; my list of commits is growing unwieldy!

### CODE REVIEW

**Comment by zbay** [comment]
@lyonsa

File: _posts/2014-03-19-zachAllenMarchMadness.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+We misinterpreted the instructions and originally operated off of one branch entitled "ZachAllen". Zach wrote the fields for Mercer, Allen wrote the 
+fields for Dayton, and we pushed from our local branches to the master. Then we pulled from the master and filled in the info 
+(wins, appearances, and coach).
+
+We had to start over with the two separate branches to comply with the assignment, Mercer and Dayton. The complexity of trying to undo what we had 
+done caused some problems, including merge conflicts that had to be resolved in Nitrous.There was some nonsensical code added to the Dayton file when 
+it was committed. Deleting it somehow fixed the merge conflict.
+
+We will chalk our difficulties up to an inadequate grasp on the branching paradigm. Using a single branch for our edits seemed like a reasonable 
+idea at first. It is just now setting in, at least for Zach, that branching  must be done liberally and branches updated only from the master branch.
```
Comment (eah13) [inline _posts/2014-03-19-zachAllenMarchMadness.md:22]: Nice!  That's the key realization

**Comment by eah13** [comment]
Please link to the two merged pull requests you used for this assignment within your post.  (i.e. evidence you did the work :)

**Comment by zbay** [comment]
Added the links.

**Comment by eah13** [comment]
Nice! Sorry I missed this fix.

**Pull request merged**